tools/sense_studio/annotation.py

```diff
@@ -112,20 +110,19 @@ def annotate(project, split, label, idx):
     videos = os.listdir(frames_dir)
     videos.sort()
 
-    features = np.load(os.path.join(features_dir, f'{videos[idx]}.npy'))
-    features = features.mean(axis=(2, 3))
+    # The list of images in the folder
+    images = [image for image in glob.glob(os.path.join(frames_dir, videos[idx], '*'))
+              if utils.is_image_file(image)]
 
     # Load logistic regression model if available
     logreg_path = os.path.join(logreg_dir, 'logreg.joblib')
-    if os.path.isfile(logreg_path):
+    features_path = os.path.join(features_dir, f'{videos[idx]}.npy')
+    if os.path.isfile(logreg_path) and os.path.isfile(features_path):
         logreg = load(logreg_path)
+        features = np.load(os.path.join(features_dir, f'{videos[idx]}.npy')).mean(axis=(2, 3))
         classes = list(logreg.predict(features))
```

Comment:
Shouldn't this part only be performed, if the setting is enabled?

Reply:
Updated this to use the project-setting